lib/devise/controllers/helpers.rb

```diff
@@ -210,6 +210,7 @@ def signed_in_root_path(resource_or_scope)
       #       else
       #         super
       #       end
+      #     end
```

Comment:
there's no need for that `end`. We need one `end` for `def`  and one for `if`

Reply:
Ah, I confused `||` with the beginning of code block. Sorry for the noise.